app-misc/hello/hello-2.10.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="GNU Hello program"
+DESCRIPTION="The GNU Hello program produces a familiar, friendly greeting.\
+Yes, this is another implementation of the classic program that prints \"Hello, world!\" when you run it.\
+However, unlike the minimal version often seen, GNU Hello processes its argument list to modify its behavior,\
```

Comment:
Suggestion:
```diff
-SUMMARY="GNU Hello program"
+SUMMARY="An implementation of the \"Hello World!\" program"
-DESCRIPTION="The GNU Hello program produces a familiar, friendly greeting.\
-Yes, this is another implementation of the classic program that prints \"Hello, world!\" when you run it.\
-However, unlike the minimal version often seen, GNU Hello processes its argument list to modify its behavior,\
-supports greetings in many languages, and so on.\
-The primary purpose of GNU Hello is to demonstrate how to write other programs that do these things;\
+DESCRIPTION="The GNU Hello program produces a familiar, friendly greeting. \
+Yes, this is another implementation of the classic program that prints \
+\"Hello, world!\" when you run it. However, unlike the minimal version often \
+seen, GNU Hello processes its argument list to modify its behavior, supports \
+greetings in many languages, and so on. \
+The primary purpose of GNU Hello is to demonstrate how to write other programs \
+that do these things; \
 it serves as a model for GNU coding standards and GNU maintainer practices."
```

Reply:
I forgot to say that this is the same text as you had, except that I folded the lines differently, in order to respect the recommended 80 chars limit (per line) in this part of the recipe.